src/document/DocumentAnnotator.ts

```diff
@@ -78,4 +86,50 @@ export default class DocumentAnnotator extends BaseAnnotator {
 
         this.managers.set(pageNumber, pageManagers);
     }
+
+    scrollToAnnotation(annotationId: string | null): void {
+        if (!annotationId) {
+            return;
+        }
+
+        const annotation = getAnnotation(this.store.getState(), annotationId);
+        const annotationPage = annotation?.target.location.value;
+
+        if (!annotation || !annotationPage) {
+            return;
+        }
+
+        if (isRegion(annotation)) {
+            this.scrollToLocation(annotationPage, centerRegion(annotation.target.shape));
+        }
+    }
+
+    scrollToLocation(page = 1, { x: offsetX, y: offsetY } = { x: 0, y: 0 }): void {
+        const pageEl = this.getPage(page);
+
+        if (!this.annotatedEl || !pageEl) {
+            return;
+        }
+
+        const canSmoothScroll = 'scrollBehavior' in this.annotatedEl.style;
```

Comment:
This variable seems only used once.

Reply:
Yeah, but the variable name helps describe its purpose better than the code does, I think.